presto-docs/src/main/sphinx/security/file-based-system-access-control.rst

```diff
@@ -0,0 +1,93 @@
+================================
+File Based System Access Control
```

Comment:
Call this just "System Access Control", and explain in the text a) what a SystemAccessControl plugin is (broadly speaking), and b) that you can either use a custom one or the built-in file-based implementation.